app/src/main/res/values/strings.xml

```diff
@@ -4,7 +4,6 @@
 
     <string name="pokemon029">Nidoran♀</string>
     <string name="pokemon032">Nidoran♂</string>
```

Comment:
Can we also remove `pokemon029` and `pokemon032` like `pokemon133 `, since they're also redundant?